src/test/java/com/missafternoon/budjen/transactions/ConverterTest.java

```diff
@@ -0,0 +1,75 @@
+package com.missafternoon.budjen.transactions;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConverterTest {
+
+    @Test
+    public void newTransactionCreated() {
+        String[] args = new String[]{"add", "--debit", "--description", "fish", "--amount", "500"};
+        Transaction transaction = new Converter().convert(args);
+        Assert.assertNotNull(transaction);
+        assertEquals(TransactionType.DEBIT, transaction.getType());
+        assertEquals("fish", transaction.getDescription());
+        assertEquals(500, transaction.getMoney());
+    }
+
+    @Test
+    public void inputInDifferentOrders_stillCreatesNewTransaction() {
+        String[] args = new String[]{"add", "--amount", "2300", "--description", "refund", "--credit"};
+        Transaction transaction = new Converter().convert(args);
+        assertEquals(2300, transaction.getMoney());
+        assertEquals("refund", transaction.getDescription());
+        assertEquals(TransactionType.CREDIT, transaction.getType());
+    }
+
+}
+
+class Converter {
+
+    Transaction convert(String[] args) {
+        return makeTransaction(args);
```

Comment:
This public method, and the private method below have the same signature (input arguments and return value), and this method doesn't do anything else but delegate to the private method, so we might as well inline this method call and get rid of `makeTransaction`.